package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
+    "klicker-knight": "ts-node index.ts"
+  },
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
```

Comment:
Should we use node16?

Reply:
Should probably just set it to 18

Reply:
I'm fine with 14. Can you set the `engines` property?

Reply:
Updated everything to 16